### PR TITLE
Update events-framework docs

### PR DESCRIPTION
**Pull request by tpiperatgod**
Signed-off-by: laminar <fangtian@kubesphere.io>

### CODE REVIEW

File: docs/concepts/OpenFunction-events-framework.md [hunk 1]
```diff
@@ -72,19 +72,45 @@ In this sample, the event source is a Kafka server and the target function is a
...
+      runtime: "knative"
+      template:
+        containers:
+          - name: function
+            imagePullPolicy: Always
+  ```
+
+  You can get the URL address of the function with the following command:
+
+  > Please note that when you use this URL, you need to extend it to the following style:
```
Comment (benjaminhuo) [inline docs/concepts/OpenFunction-events-framework.md:96]: `openfunction.io` is ok for most of the cases and so does `openfunction.io.svc`: 

https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/ 
`In summary, a pod in the test namespace can successfully resolve either data.prod or data.prod.svc.cluster.local`

User might customize cluster.local to another customized domain suffix on the contrary.

Just mention `openfunction` is a service in namespace `io` is all right.

File: docs/concepts/OpenFunction-events-framework.md [hunk 2]
```diff
@@ -206,95 +233,106 @@ function-sample-serving-qrdx8-ksvc-fwml8-v100-deployment-8cpxsj   2/2     Runnin
 
 - A Knative runtime function (target function)
 
-  You can refer to [Sample Function Go](https://github.com/OpenFunction/samples/tree/main/functions/Knative/hello-world-go) to create a Knative runtime function.
+  You can apply the following configuration to create a function that will be used to be triggered.
```
Comment (benjaminhuo) [inline docs/concepts/OpenFunction-events-framework.md:236]: =>
You can create a function like below as the target function to trigger

File: docs/concepts/OpenFunction-events-framework.md [hunk 3]
```diff
@@ -206,95 +233,106 @@ function-sample-serving-qrdx8-ksvc-fwml8-v100-deployment-8cpxsj   2/2     Runnin
...
+        containers:
+          - name: function
+            imagePullPolicy: Always
+  ```
 
-  Here I assume that the name of this function (Knative Service) is `function-sample-serving-ksvc` . You should be able to see this value with the `kubectl get ksvc` command
+  You can get the URL address of the function with the following command:
 
-- A Kafka server (event source)
+  > Please note that when you use this URL, you need to extend it to the following style:
```
Comment (benjaminhuo) [inline docs/concepts/OpenFunction-events-framework.md:257]: Same here

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: docs/concepts/OpenFunction-events-framework.md (hunk 1 above)
Comment (tpiperatgod) [reply docs/concepts/OpenFunction-events-framework.md:96]: ok

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**